opencog/moses/deme/deme_expander.cc

```diff
@@ -489,7 +489,7 @@ void deme_expander::optimize_demes(int max_evals, time_t max_time)
             // Optimize
             if (_params.atomspace_port) {
                 ComboToAtomeseConverter _to_atomese;
```

Comment:
Putting `_` in front of a variable is usually reserved to class members (and even more usual private members).

Reply:
Also, the converter needs to be constructed with an atomspace, the same atomspace that has been used to populate the features `(SchemaNode "1")`, etc. Otherwise the values will be attached to the schema nodes, and thus the atomese interpreter will fail to evaluate programs using those.